src/inc/til/env.h

```diff
@@ -313,6 +313,15 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 
                             if (valueNameSize)
                             {
+                                bool isPathVar = is_path_var(valueName);

```

Comment:
```suggestion
                                const bool isPathVar = is_path_var(valueName);
```

apparently static analysis got mad at this one